test/python/quantum_info/operators/test_random.py

```diff
@@ -190,6 +190,26 @@ def test_not_global_seed(self):
         rng_after = np.random.randint(1000, size=test_cases)
         self.assertFalse(np.all(rng_before == rng_after))
 
+    def test_cliffords_2q(self):
+        """Test that we get all 2-qubit Cliffords (actually symplectic
+        matrices) with sufficiently many trials.
+        """
+        seen = set()
+        for seed in range(10000):
```

Comment:
Are 10000 trials negligible in terms of run time in this case? We want to avoid unit tests taking too much time. Perhaps using one-time preprocessing it's possible to find a small set of seeds such that a small number of iterations on each one is enough to guarantee full coverage?

Reply:
Yes, this is fast -- the full test takes 0.17 seconds on my laptop. Regarding your suggestion, we probably don't want to update the test each time that we slightly change the algorithm. Of course, there is a tiny chance that after a certain hypothetical change even 10,000 random Cliffords would not be enough to hit all of 720 symplectic matrices, however for a true random Clifford generation algorithm this chance would be bounded above by $720 * (719/720)^{10000}$, which is 0.067%. In other words, if the test does fail after a future hypothetical change, it would most probably point to a bug in the code.